x-pack/plugins/search_indices/public/components/quick_stats/quick_stat.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiAccordion,
+  EuiDescriptionList,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPanel,
+  EuiText,
+  useEuiTheme,
+  useGeneratedHtmlId,
+} from '@elastic/eui';
+
+interface BaseQuickStatProps {
+  icon: string;
+  iconColor: string;
+  title: string;
+  secondaryTitle: React.ReactNode;
+  open: boolean;
+  content?: React.ReactNode;
+  stats: Array<{
+    title: string;
+    description: NonNullable<React.ReactNode>;
+  }>;
+  setOpen: (open: boolean) => void;
+  first?: boolean;
+}
+
+export const QuickStat: React.FC<BaseQuickStatProps> = ({
+  icon,
+  title,
+  stats,
+  open,
+  setOpen,
+  first,
+  secondaryTitle,
+  iconColor,
+  content,
+  ...rest
+}) => {
+  const { euiTheme } = useEuiTheme();
+
+  const id = useGeneratedHtmlId({
+    prefix: 'formAccordion',
+    suffix: title,
+  });
+
+  return (
+    <EuiAccordion
+      forceState={open ? 'open' : 'closed'}
+      onToggle={() => setOpen(!open)}
+      paddingSize="none"
+      id={id}
+      buttonElement="div"
+      arrowDisplay="right"
+      {...rest}
+      css={{
+        borderLeft: euiTheme.border.thin,
```

Comment:
would we want this in sass? or emotion? I'm not sure what the preferred option is for customizing CSS. @sphilipse ?